beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -1,303 +1,247 @@
 # -*- coding: utf-8 -*-
-
 """
 `Introduction to ONNX <intro_onnx.html>`_ ||
 `Exporting a PyTorch model to ONNX <export_simple_model_to_onnx_tutorial.html>`_ ||
-**Extending the ONNX Registry**
+**Extending the ONNX exporter operator support** ||
+`Export a model with control flow to ONNX <export_control_flow_model_to_onnx_tutorial.html>`_
 
-Extending the ONNX Registry
-===========================
+Extending the ONNX Exporter Operator Support
+============================================
 
-**Authors:** Ti-Tai Wang (titaiwang@microsoft.com)
+**Authors:** Ti-Tai Wang (titaiwang@microsoft.com), Justin Chu (justinchu@microsoft.com)
 """
 
 
 ###############################################################################
 # Overview
 # --------
 #
-# This tutorial is an introduction to ONNX registry, which empowers users to implement new ONNX operators
-# or even replace existing operators with a new implementation.
+# This tutorial describes how you can create ONNX implementation for unsupported PyTorch operators
+# or replace existing implementation with your own.
+#
+# We will cover three scenarios that require extending the ONNX exporter's operator support:
+#
+# * Overriding the implementation of an existing PyTorch operator
+# * Using custom ONNX operators
+# * Supporting a custom PyTorch operator
+#
+# Overriding the implementation of an existing PyTorch operator
```

Comment:
Can you also add a What you will learn section. Example:

In this tutorial, you will learn:

* How to override or add support for PyTorch operators in ONNX.
* How to integrate custom ONNX operators for specialized runtimes.
* How to implement and translate custom PyTorch operators to ONNX.

Reply:
Done